### PR TITLE
feat: tooltip placement for scroll and left/right

### PR DESCRIPTION
**Pull request by NatalieLoyzaga**
# Objective

- Add left and right placement variant. 
- Allow the popper to 'flip' depending on the space within the viewport -> see Scroll story example.

# Motivation and Context

- Match the UI kit.

# Screenshots (if appropriate)
image
image

### CODE REVIEW

**Comment by jmitsios** [comment]
Awesome stuff Nat :) 

Could I suggest we reduce the distance between the tooltip and the asset to **6px** please?   I believe the spacing in the UI Kit is **12px**, but it's measuring it from the base of the _container_ (bubble) rather than the _tip_.

This may also help in the [Sticker Sheet](https://dev.cultureamp.design/KDS-70/Tooltip-placement/storybook/?path=/story/components-tooltip--sticker-sheet) layout, which currently has all the tooltips a touch too close to all the surrounding buttons.

File: draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.scss [hunk 1]
```diff
@@ -83,6 +83,19 @@ $arrow-width: 20px;
...
     }
   }
+
+  [data-popper-placement^="left"] & {
+    right: 0;
+    .arrowInner {
+      transform: rotate(270deg);
+    }
+  }
+  [data-popper-placement^="right"] & {
```
Comment (mcwinter07) [inline draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.scss:93]: I didn't know about the `^=` selector, very handy

**Review by mcwinter07 — COMMENTED** [review]
Looks pretty good to me. Just a quick query about `rtl` and `ltr` styling - it's to affect the placement but only on the sticker sheet - just want to make sure that's either a weird storybook outlier or intended?

**Review by mcwinter07 — APPROVED** [review]
I like the update to the sticker sheet 👍  Good to merge

**Pull request merged**